types/astring/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for astring 1.3
+// Project: https://github.com/davidbonnet/astring
+// Definitions by: Nikolaj Kappler <https://github.com/nkappler>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as ESTree from 'estree';
+import "node";
+import { Stream } from 'stream';
+
+declare namespace astring {
```

Comment:
Get rid of the namespace, export = statement and move everything inside the namespace in the outer scope. (directly exporting)